File: pattern.js
let input = "welcome";
let mid= Math.floor(input.length/2)
let newInput = input.substring(mid) + input.substring(0,mid);
let printText = "";
for(let i=0;i<newInput.length;i++) {
  printText += newInput.charAt(i);
  console.log(printText);
}

File: pattern.java
	public static void main(String[] args) {
		String input = "WELCOME";
		String newInput = input.substring(input.length()/2) + input.substring(0,input.length()/2);
		String printText = "";
		for(int i=0;i<newInput.length();i++) {
		    printText += newInput.charAt(i);
		    for(int j=0;j<newInput.length()-i;j++)
		        System.out.print(" ");
		    System.out.print(printText);
		    System.out.println("");
		}
	}
